examples/volumes/vsphere/vsphere-volume-sc-vsancapabilities.yaml

```diff
@@ -0,0 +1,9 @@
+kind: StorageClass
+apiVersion: storage.k8s.io/v1beta1
+metadata:
+  name: fast
+provisioner: kubernetes.io/vsphere-volume
+parameters:
+    diskformat: zeroedthick
+    hostFailuresToTolerate: "2"
```

Comment:
I suggest in this iteration, let's make datastore param mandatory. We can revisit and make it optional when we add support for specifying SPBM policy or when VCS has ability specify default datastore for K8S cluster like vDVS.